style.css

```diff
@@ -245,6 +223,84 @@ h3 {
   cursor: pointer;
 }
 
+/* The contact form */
+.form-content {
+  padding: 24px;
+}
+
+.form-content h2 {
+  color: #9093ff;
+  padding: 0;
+  font-size: 32px;
+  text-align: center;
+  margin-top: 50px;
+  margin-left: 20px;
+  font-weight: 500;
+}
+
+.form-content p {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f5f5ff;
+  margin-top: 24px;
+  margin-bottom: 40px;
+  width: 327px;
+  margin-left: 35px;
+  font-weight: 400;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  height: 48px;
+  width: 100%;
+  margin: 12px 0;
+}
+
+#text-message {
+  height: 148px;
+  width: 100%;
+  margin: 12px 0;
+}
+
+#button {
+  margin: 30px 0;
+}
```

Comment:
- [ ] Your buttons texts are not centered too. Try using `justify-content` to position the text inside the button or set a `padding` for it to look like the Figma template.
 -
<img width="423" alt="image" src="https://user-images.githubusercontent.com/61919617/186195553-3dfdac1d-63b4-40e3-97de-ed969e75aac2.png">